vunit/ui.py

```diff
@@ -1023,6 +1023,9 @@ def get_implementation_subset(self, source_files):
         return SourceFileList([SourceFile(source_file, self._project, self)
                                for source_file in source_files])
 
+    def get_builtins(self):
```

Comment:
I do not want to make the Builtins class public. Just add one method to get the built_in_libraries directly.